kuma/static/js/components/payments/payments-handler.js

```diff
@@ -126,10 +98,12 @@
 
     // Ensure we only show the form if js is enabled
     if (win.StripeCheckout) {
-        $('#contribution-popover-container').removeClass('is-hidden');
+        $('#mdn-payments').removeClass('hidden');
```

Comment:
I'm assuming that most of the changes in this file are the result of prettier reformating things.